<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditAnnotations</class>
 <widget class="QDialog" name="DialogEditAnnotations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit annotations</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listAll">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_edit">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_text">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Text:&lt;/p&gt;&lt;p&gt;(can be HTML)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPlainTextEdit" name="textEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_size">
           <property name="text">
            <string>Text size (px):</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinSize">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_location">
           <property name="text">
            <string>Location (x,y):</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QFrame" name="frame_location">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSpinBox" name="spinX">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinY">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_buttons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btUp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btDown">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btDuplicate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Duplicate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_buttonbox">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btOk</sender>
   <signal>clicked()</signal>
   <receiver>DialogEditAnnotations</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogEditAnnotations</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
